notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>We should definitely be keeping the outputs in the checked in notebooks. We're going to potentially embed this into the docs so that users can still go through the notebooks even without running them. Can you also add a cell at the beginnign (maybe under the imports cell) to print <code>nvidia-smi</code>?This way the user has information about the gpu and architecture this was executed on. </p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Thanks, I'm adding the <code>nvidia-smi</code> output; but I'm not sure what do you mean regarding keeping the outputs? Would you like me to print the neighbors/distances? </p>

Reply:
<p>I just mean running the notebook and checking in the notebook w/ the outputs. Some libraries do a "clear outputs" before they check the notebooks in but I find having the outputs is useful to users, especially if they don't want to run the notebooks themselves. </p>